prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -208,7 +190,17 @@ presubmits:
       containers:
       - image: python:3.7
         command:
-        - ./v2/test/presubmit.sh
+        - ./v2/test/sample-test.sh
+  
+  - name: kubeflow-pipelines-integration-v2
+    run_if_changed: "^(samples/core/(parameterized_tfx_oss|dataflow)/.*)$"
```

Comment:
Shouldn't we enable this test at all time? Since it is testing the integration of sample/backend/sdk/etc.

Reply:
Well, the test name is actually misleading, the test itself only tests dataflow and tfx samples and these samples typically take a very long time to execute, so we name the test integration -- "integration with other non-KFP services".

The actual integration test of sample/backend/sdk/etc is kfp e2e test.

Reply:
Thank you for the clarification! It makes sense to me.